src/js/components/bulkDownload/awards/filters/AgencyFilter.jsx

```diff
@@ -129,16 +128,15 @@ export default class AgencyFilter extends React.Component {
         ));
 
         // Create the sub-agency options
-        const subAgencies = this.props.subAgencies.map((subAgency) => (
+        const subAgencies = this.props.subAgencies.map((subAgency, index) => (
             <li
                 className="field-item"
-                key={`field-${subAgency.subtier_agency_id}`}>
+                key={`field-${index}`}>
```

Comment:
Are subtier agency names unique? If so, can you use that instead of `index` as the key?